autoagents/agents/agents/search.py

```diff
@@ -166,15 +206,19 @@ class ActionRunner:
     def __init__(self,
                  outputq,
                  llm: BaseLanguageModel,
-                 persist_logs: bool = False):
+                 persist_logs: bool = False,
+                 prompt_template: str = template,
+                 tools: List[Tool] = [search_tool, note_tool, finish_tool],
+                 search_tool_name: str = "Search"):
         self.ialogger = InteractionsLogger(name=f"{uuid.uuid4().hex[:6]}", persist=persist_logs)
         tools = [search_tool, note_tool, finish_tool]
```

Comment:
This line can be removed because it overwrites the `tools` variable.

Reply:
fixed